index.css

```diff
@@ -470,3 +480,386 @@ textarea {
   border-radius: 10px;
   margin-bottom: 20px;
 }
+
+/* Desktop version */
+
+.nav-items {
+  display: none;
+}
+
+.border-right {
+  display: none;
+}
+
+.content-image-desktop {
+  display: none;
+}
+
+.project-card .desktop-title {
+  display: none;
+}
+
+.content-desktop {
+  display: none;
+}
+
+@media screen and (min-width: 992px) {
+  * {
+    margin: 0;
+    padding: 0;
+  }
+
+  /* Nav-bar */
+
+  .menu-icon {
+    display: none;
+  }
+
+  .social li {
+    margin-bottom: 15px;
+  }
+
+  .nav-items ul li {
+    margin-right: 32px;
+  }
+
+  .name {
+    margin-top: 0;
+    margin-left: 50px;
+  }
+
+  .nav-bar {
+    margin-bottom: 20px;
+    margin-top: 20px;
+  }
+
+  .nav-items {
+    margin-right: 70px;
+    display: flex;
+    font-family: "Inter", Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .nav-items ul {
+    display: flex;
+  }
+
+  /* Welcome section */
+
+  .welcome-section {
+    height: 100vh;
+    background-image: url(./images/Illustration-header-desktop-2.png);
+    background-repeat: no-repeat;
+    background-attachment: scroll;
+    background-position: 604px -70px;
+    background-color: #1c1a19;
+    display: flex;
+    flex-direction: row;
+    padding: 109px 24px;
+  }
```

Comment:
- [ ] [OPTIONAL] I have checked the background for the welcome section on the desktop version and as you can see on the screenshot below it is not cover the whole screen. Your project is approved, but kindly ask you to fix this issue before merging. You can do it by giving `background-position: ` 100%
 
> Your current welcome section
<img width="1422" alt="Screen Shot 2022-02-25 at 10 24 21" src="https://user-images.githubusercontent.com/49064106/155665476-5119fa0d-74b8-441c-bc0f-8dfd28efb6c6.png">

> After me set `background-position: 100%;`
<img width="1431" alt="Screen Shot 2022-02-25 at 10 25 09" src="https://user-images.githubusercontent.com/49064106/155665570-7afdb362-c0d2-421f-8b6b-2fbc0bab9daa.png">